### PR TITLE
fix(wizard): fixes multilingual/multispeaker config setting in new projects

### PR DESCRIPTION
**Pull request by wiitt**
Fixes https://github.com/EveryVoiceTTS/EveryVoice/issues/470.

### PR Goal? 
This PR fixes multilingual/multispeaker config setting for new projects.

### Fixes? 
Fixes https://github.com/EveryVoiceTTS/EveryVoice/issues/470.

### Feedback sought? 

1. Check the correctness of the commit history. The branch for this PR has been rebased on top of the PR https://github.com/EveryVoiceTTS/EveryVoice/pull/509 branch. I wonder if I should still request merge into main or into a different branch.
2. Feedback on wording of new prompts in the wizard is greatly appreciated.
3. Take a close look at test_wizard.py/WizardTest/test_running_out_of_columns. It has steps added to make the test case complete.  I though that it would be better if the test would go through all steps and compare output with the expected one. Is it appropriate for this test or was it testing some other aspect rather work of the tour?

### Priority? 
Medium.

### Tests added? 
Modified the complete-tour wizard tests to match new wizard workflow. Added a new complete-tour wizard test with two datasets to check correctness of multilingual/multispeaker marking.

### How to test? 
Run the wizard and create projects with various combinations of datasets (same language/same speaker; same language/different speakers; different languages/same speaker; different language/different speakers; with a specified language/speaker column). Try out different answers to two new questions.

### Confidence? 
Medium. Big update.

### Version change? 
Can be included in a new release with more important fixes.  

### Related PRs? 
PR https://github.com/EveryVoiceTTS/EveryVoice/pull/509 should be merged first.

### CODE REVIEW

**Comment by wiitt** [comment]
Made the last forced push to cover with testing two additional lines.

**Comment by wiitt** [comment]
@roedoejet I've changed the branch to merge into to the branch from https://github.com/EveryVoiceTTS/EveryVoice/pull/509.

**Review by roedoejet — CHANGES_REQUESTED** [review]
I think we should refer to it as a speaker ID instead of name, but this looks great so far, with extensive testing. I will test it out on my machine now but just wanted to post some of these text requested changes in the meantime. Thanks @wiitt !

File: everyvoice/wizard/dataset.py [hunk 1]
```diff
@@ -497,6 +500,9 @@ def validate(self, response):
         return response in self.choices
 
     def effect(self):
+        print(
+            "Note: if your dataset has speakers with names matching with speakers from other provided datasets, they will be considered the same. If it is not a desired behaviour, you will have to alter speaker names to indicate that they are different."
```
[the remark this answers is not in the document]
Comment (wiitt) [reply everyvoice/wizard/dataset.py:504]: Do we keep "speaker names" here?

**Review by roedoejet — APPROVED** [review]
lgtm!

**Pull request merged**